packages/form/src/Select/Select.js

```diff
@@ -51,11 +52,13 @@ class Select extends Component<Props> {
       className,
       style,
       required,
+      disabled,
       placeholder,
       options,
       onChange,
       value,
       loading,
+      boolean,
```

Comment:
Wat is het nut van de `boolean` prop. Deze lijkt nergens te worden gebruikt?

Reply:
Aha klein foutje, ik los het op :)